DataFormats/SiStripCluster/src/classes_def.xml

```diff
@@ -24,8 +24,8 @@
  <class name="edm::Wrapper<edmNew::DetSetVector<edm::Ref<edmNew::DetSetVector<SiStripCluster>,SiStripCluster,edmNew::DetSetVector<SiStripCluster>::FindForDetSetVector> > >" />
 
 
- <class name="SiStripApproximateCluster" ClassVersion="3">
-  <version ClassVersion="3" checksum="2041370183"/>
+ <class name="SiStripApproximateCluster" ClassVersion="4">
+  <version ClassVersion="4" checksum="2854791577"/>
```

Comment:
This should have kept the class version for version 3.

Reply:
Thanks Matti. Is there a way to detect this at compile time?

Reply:
Nothing straightforward comes to my mind, but maybe we could figure out something. A specific check would certainly help to catch such cases. Could you open an issue about it?

Reply:
done! https://github.com/cms-sw/cmssw/issues/39008